client/src/components/atoms/RichTextField.tsx

```diff
@@ -52,19 +53,27 @@ const RichTextField: FunctionComponent<Props> = (props: Props) => {
             setActive(true)
         }
 
+        let emptyContent = false;
         // check if we are already typing (we are active) and state has no text
         if (active && (!state.getCurrentContent().hasText() && state.getCurrentContent().getFirstBlock().getType() === 'unstyled')) {
             // if so, we made the content empty, so add back the default text
             setActive(false)
+            emptyContent = true;
```

Comment:
Perhaps just 

if (props.onEmpty) { props.onEmpty() } 

here

Reply:
wanted to avoid firing `props.onEmpty()` and `props.onChange(JSON.stringify(convertToRaw(state.getCurrentContent())));` because it may cause a race condition

(assuming that  the parent component will not use check that `JSON.stringify(convertToRaw(state.getCurrentContent()))` is empty or not)